From 1436ea2b03a9b86a8706bd2e001550221914cb2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lubom=C3=ADr=20Sedl=C3=A1=C5=99?= <lsedlar@redhat.com>
Date: Wed, 4 Apr 2018 12:23:43 +0200
Subject: [PATCH] tests: Use dummy modulesdir for DNF
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Otherwise it tries to ensure it exists, and since the default is
/etc/dnf/modules.d, it's causing problems if the directory does not
exist and user does not have permissions to create it.

Signed-off-by: Lubomír Sedlář <lsedlar@redhat.com>
---
 tests/test_gather.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_gather.py b/tests/test_gather.py
index 1daf3c60..25dfc399 100644
--- a/tests/test_gather.py
+++ b/tests/test_gather.py
@@ -1810,6 +1810,7 @@ class DNFDepsolvingTestCase(DepsolvingBase, unittest.TestCase):
         conf = Conf(base_arch)
         conf.persistdir = persistdir
         conf.cachedir = self.cachedir
+        conf.modulesdir = os.path.join(persistdir, 'modules.d')
         if exclude:
             conf.exclude = exclude
         dnf = DnfWrapper(conf)
-- 
2.13.6

